docs/guides/react-server-cli.md

```diff
@@ -157,7 +157,8 @@ export default (server, reactServerMiddleware) => {
 	server.use(compression());
 	server.use(bodyParser.urlencoded({ extended: false }));
 	server.use(bodyParser.json());
-	server.use(session(options));
+	server.use(helmet());
```

Comment:
well...we are because of #812, but we missed updating this example to depict the new default express middleware.